actors/builtin/market/market_test.go

```diff
@@ -350,3 +439,135 @@ func (h *marketActorTestHarness) expectProviderControlAddressesAndValidateCaller
 		exitcode.Ok,
 	)
 }
+
+func (h *marketActorTestHarness) withdrawStorageMinerBalanceOK(rt *mock.Runtime, owner, worker, provider address.Address, withDrawAmt, expectedSend abi.TokenAmount) {
+	rt.SetCaller(worker, builtin.AccountActorCodeID)
+	h.expectProviderControlAddressesAndValidateCaller(rt, provider, owner, worker)
+
+	params := market.WithdrawBalanceParams{
+		ProviderOrClientAddress: provider,
+		Amount:                  big.Add(withDrawAmt, big.NewInt(1)),
```

Comment:
This is a weird thing to have inside the harness method. We really shouldn't be doing anything tricky here. Could you move this out to the call site/s for which it's relevant?